extra-tests/tests/Logger_Tests_ExperienceSite.cls

```diff
@@ -9,12 +9,23 @@
  */
 @IsTest
 private class Logger_Tests_ExperienceSite {
+    private static final Boolean IS_EXPERIENCE_CLOUD_ENABLED = Schema.getGlobalDescribe().containsKey('Network');
```

Comment:
nice

Reply:
Yeah, I don't typically enable experience cloud/communities in my "local" scratch orgs, so these tests were previously always failing, and I got tired of that 😆